### ISSUE TITLE
Support for searx.be

### ISSUE THREAD

**Issue by emanuelegori** [Fivefold/linkding-injector#27]
searx.be respects the recommended syntax

ex: https://searx.be/search?q=Self-hosted+start+page+alternatives

any suggestions ?

**Comment by Fivefold** [comment]
Hey @emanuelegori, please make sure you are using the "simple" theme. You can check in the preferences of searx, found in the top right corner of the page:

![image](https://github.com/Fivefold/linkding-injector/assets/38764218/1531e24f-5f01-4bea-8002-06bd82f14c23)

Other themes than the "simple" theme are unfortunately not supported at the moment and may or may not work.

Does this solve your issue (at least with the "simple" theme)?

**Comment by emanuelegori** [comment]
yes, even if some instances, even if configured as you suggested, still don't work.

ex.: https://searxng.devol.it/   it does not work
       https://priv.au work          this works

I support your work. Thank you

**Comment by Fivefold** [comment]
Thank you, I see the issue. 
The devol.it instance does not follow the required [URL pattern](https://github.com/Fivefold/linkding-injector/wiki/SearX-SearXNG-support) and thus the injection script is not inserted. I might be able to make this more robust in the future to handle more variants of SearX, but due to my current workload it's not a priority.

For the time being please use other SearX instances or consider hosting your own.
